docs/tutorials/aws.md

```diff
@@ -80,18 +81,53 @@ spec:
     spec:
       containers:
       - name: external-dns
-        image: registry.opensource.zalan.do/teapot/external-dns:v0.4.2
+        image: registry.opensource.zalan.do/teapot/external-dns:v0.4.7
         args:
         - --source=service
         - --source=ingress
         - --domain-filter=external-dns-test.my-org.com # will make ExternalDNS see only the hosted zones matching provided domain, omit to process all available hosted zones
         - --provider=aws
         - --policy=upsert-only # would prevent ExternalDNS from deleting any records, omit to enable full synchronization
+        - --aws-zone-type=public #only look at public hosted zones (valid values are public/private)
         - --registry=txt
         - --txt-owner-id=my-identifier
 ```
 
-## Verify ExternalDNS works
+## Arguments
+
+This list is not the full list, but a few arguments that where choosen.
+
+### aws-zone-type
+
+`aws-zone-type` allows filtering for private and public zones
+
+
+## Verify ExternalDNS works (Ingress example)
+Change your ingress resource manifest file. In example below, I am using nginx ingress controller
+
+```yaml
+
+apiVersion: extensions/v1beta1
+kind: Ingress
+metadata:
+  name: foo
+  namespace: dev
+  annotations:
+    kubernetes.io/ingress.class: "nginx"
+    ingress.kubernetes.io/ssl-redirect: "true"
+    ingress.kubernetes.io/force-ssl-redirect: "true"
+    external-dns.alpha.kubernetes.io/hostname: foo.bar.com
```

Comment:
`external-dns.alpha.kubernetes.io/hostname: foo.bar.com` is not needed for external-dns as it will use the value from the spec. I would leave it out of the simple example.

Reply:
wait.. so you dont need the annotation!?

Reply:
@shavo007 In `Ingress` you don't need it as it's down in the rules (unless you don't have any rules and rely on the `defaultBackend` behaviour). For `Services` you always need it (unless you run ExternalDNS with the `--fqdn-template` option which gives you a default for each service).